### ISSUE TITLE
Improve Button names and labels for screen readers

### ISSUE THREAD

**Issue by MokaCake** [CMU-313/Teedy#262]
Teedy scored 69 on accessibility on Light house with 18 passing audits and 18 not applicable ones. One failing element in the test is that some buttons do not have an accessible name, which results in confusion because the screen reader will simply call all of them buttons, and therefore cause confusion for users who rely on screen readers and therefore make Teedy less (or even not) accessible for them 
<img width="184" alt="Screen Shot 2022-09-07 at 11 09 38 PM" src="https://user-images.githubusercontent.com/71565937/189025495-b2a077a6-6203-4d87-86ff-8d254efe8856.png">
<img width="525" alt="Screen Shot 2022-09-07 at 11 09 32 PM" src="https://user-images.githubusercontent.com/71565937/189025508-d9ef6311-a2d5-4ee8-9005-c0f81ef64aa5.png">
<img width="450" alt="Screen Shot 2022-09-07 at 11 09 25 PM" src="https://user-images.githubusercontent.com/71565937/189025517-772f6924-ddfa-48ec-9f01-7b2499b2ffc4.png">
In order to improve this metric, I am planning on finding more descriptive names for these buttons so that this webpage is more accessible to everyone.

**Comment by MokaCake** [comment]
I would like to work on this please!
